### ISSUE TITLE
linter: run oxlint cli only got warnings but no errors

### ISSUE THREAD

**Issue by rendeyuwei** [oxc-project/oxc#6619]
### What version of Oxlint are you using?

0.9.10

### What command did you run?

npx oxlint

### What does your `.oxlint.json` config file look like?

not using a config file

### What happened?

when i run oxlint cli, it only throw warning, but no errors. is it normal ?thx
![image](https://github.com/user-attachments/assets/ebd9c703-7f05-4b42-b5fa-9a4b02466fca)

**Comment by mysteryven** [comment]
It's expected behavior now,  `npx oxlint` is same with `npx oxlint -W correctness`.  If you want it to throw error, try: 

```bash
npx oxlint -D correctness
```

https://oxc.rs/docs/guide/usage/linter/cli.html#allowing-denying-multiple-lints
